tstest/integration/integration_test.go

```diff
@@ -845,7 +845,7 @@ func (n *testNode) MustUp(extraArgs ...string) {
 func (n *testNode) MustDown() {
 	t := n.env.t
 	t.Logf("Running down ...")
-	if err := n.Tailscale("down").Run(); err != nil {
+	if err := n.Tailscale("down", "--accept-risk=lose-ssh").Run(); err != nil {
```

Comment:
unrelated?

aside: should we have --accept-risk=all for stuff like this?

Reply:
probably yeah, will send separately